docs/core/animations.md

```diff
@@ -73,6 +73,16 @@ We can animate numeric attributes as well. For example, we can animate the inten
 </a-light>
 ```
 
+### Color Attributes
+
+We can animate color attributes as well. For example, we can animate a box from white to red.
```

Comment:
I'd add `We can animate any component property that has a `color` type as well.`